twins/generator_test.go

```diff
@@ -47,3 +47,33 @@ func TestPartitionSizes(t *testing.T) {
 		t.Error("did not get the expected result")
 	}
 }
+
+func TestAssignNodeIDs(t *testing.T) {
+	tests := []struct {
+		name      string
+		numNodes  uint8
+		numTwins  uint8
+		wantNodes []NodeID
+		wantTwins []NodeID
+	}{
+		{"no twins", 3, 0, []NodeID{{1, 0}, {2, 0}, {3, 0}}, nil},
```

Comment:
Should we update these to use the new API for creating replicas and twins? I'm okay with the current approach, but if keep it we should add a comment about the ordering and meaning of the replica ID vs twin ID.

Reply:
I feel adding `Replica(1)` instead of `{1,0}` makes it harder to read.
Also, is this test not enough documentation in itself? 
I feel it nicely shows the use of these.
If any, there should be documentation in the `assignNodeIDs` function.